specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3328,6 +3328,48 @@
           "nextLinkName": null
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/servicemembers/{serviceMemberId}/metrics/connectormetadata":

```

Comment:
Shouldn't {metricName} param be part of this URI?

Reply:
Fixed.